index.html

```diff
@@ -0,0 +1,175 @@
+            Listen to lectures from speakers from around the world and learn
+            about the latest trends in the world.
+          </p>
+        </div>
+        <div class="program_list">
+          <div class="program_icon">
+            <img src="img/program_icon_02.png" alt="Image"/>
+          </div>
+          <h5 class="program_title">CC Exhibition</h5>
+          <p class="program_description">
+            Listen to lectures from speakers from around the world and learn
+            about the latest trends in the world.
+          </p>
+        </div>
+        <div class="program_list">
+          <div class="program_icon">
+            <img src="img/program_icon_03.png" alt="Image"/>
+          </div>
+          <h5 class="program_title">Forum</h5>
+          <p class="program_description">
+            Listen to lectures from speakers from around the world and learn
+            about the latest trends in the world.
+          </p>
+        </div>
+        <div class="program_list">
+          <div class="program_icon">
+            <img src="img/program_icon_04.png" alt="Image"/>
+          </div>
+          <h5 class="program_title">workshop</h5>
+          <p class="program_description">
+            Listen to lectures from speakers from around the world and learn
+            about the latest trends in the world.
+          </p>
+        </div>
+        <div class="program_list">
+          <div class="program_icon">
+            <img src="img/program_icon_05.png" alt="Image"/>
+          </div>
+          <h5 class="program_title">CC Ignite</h5>
+          <p class="program_description">
+            Listen to lectures from speakers from around the world and learn
+            about the latest trends in the world.
+          </p>
+        </div>
+      </div>
+      <button type="button" class="join" id="join">
+        Join PE Summit 2022
+      </button>
+      <a href="#programs" class="see">SEE THE WHOLE PROGRAM</a>
+    </section>
+
+    <!-- Feature speaker start -->
+    <section class="speakers" id="speakers">
+      <h1>Featured Speakers</h1>
+      <div class="guidbar"></div>
+      <div class="speakerWrapper" id="speakerWrapper"></div>
+    </section>
+
+    <!-- partner section start -->
+
+    <section class="partner" id="sponser">
+      <h1 class="partner_title">Partner</h1>
+      <div class="guidbar"></div>
+      <div class="partner_logo">
+        <img src="img/14-mozilla-logo.png" alt="Mozilla-logo" />
+        <img src="img/15-google-logo.png" alt="Google-logo" />
+        <img src="img/16-naver-logo.png" alt="Naver-logo" />
+        <img src="img/17-daumkakao-logo (1).png" alt="Daumkakao-logo" />
+        <img src="img/18-Airbnb-logo.png" alt="Airbnb-logo" />
+      </div>
+    </section>
+
+    <!-- footer start -->
+    <footer class="mobile-footer">
+      <img src="img/logo.jpeg" alt="logo" />
+      <h6>2022 PAKISTAN ECOMERCE Rights Reserved</h6>
+    </footer>
```

Comment:
- [ ] These two sectios are not a strict requirement they be there on mobile version as per the Behance design. Kindly hide them on mobile view so that they are hidden and show them when display is changed to the desktop view and they are visible.